Rewamp docs content (#296)
api3dao/api3-docs: docs/curation/index.md

::: info 💡 Tip

For quick reference, you can copy-paste [`llms-full.txt`](https://docs.api3.org/llms-full.txt) to your choice of AI assistant.

:::

## Skin in the game

Unlike most curators who manage third-party capital exclusively, Api3 deploys a majority of its own treasury as the primary supplier in its vaults.
This means Api3 has direct financial exposure to every market it enables and every risk decision it makes.

External depositors are welcome to supply alongside the Api3 treasury, benefiting from the same risk management and yield strategies, with the assurance that Api3's own capital is subject to identical conditions.

## OEV advantage

Api3 is uniquely positioned as a curator because it is also the oracle provider for its own markets.
This creates a competitive advantage that no other lending protocol or curator has.

During market turmoil, liquidations on lending markets generate [Oracle Extractable Value (OEV)](/oev/) — value that the oracle has priority in capturing by batching additional operations with price updates.
On Api3-curated markets, the entirety of this OEV belongs to Api3 as protocol revenue.

> At best, other protocols receive only a smaller portion of the OEV generated on their markets.

Reviewer: This phrasing seems kinda awkward to me. What about this?
```suggestion
Other protocols, at best, capture only a fraction of the OEV generated on their markets.
```

This allows Api3 to set lower curation fees to attract borrowers, which in turn decreases the cost of borrowing.

> While this means supply-side APY is slightly lower during calm markets, it makes Api3 markets more attractive to borrowers, driving higher utilization and a healthier lending ecosystem.

Reviewer: I'd phrase this more like
```suggestion
While this means supply-side APY may be lower during calm markets, it makes Api3 markets more attractive to borrowers, driving higher utilization and a healthier lending ecosystem.
```
